Documents/submission-status.md

```diff
@@ -0,0 +1,22 @@
+## Submission status
+
+Every submission will store a `SubmissionStatus` object with the following structure:
+
+* `code`: `Integer`
+* `message`: `String`
+
+In order to track the status of a filing for a financial institution, the following states are captured by the backend:
+
+* `1`: `Created`. The filing has been created and is ready to accept data
+* `2`: `Uploading`. Data is currently being uploaded to the system
+* `3`: `Uploaded`. Data has finished uploading, is stored in the `HMDA Platform` and is ready to be checked
+* `4`: `Parsing`. The submitted information is being checked for parsing errors according to the [HMDA File Specification]().
+* `5`: `Parsed`. Filed data is conformant and is ready to be validated by the rule engine
```

Comment:
Yes, good point. This needs to be implemented in the persistence model. Created #552 for this, will update this documentation and the submission states with it in this PR